integration/update_field_compat_test.go

```diff
@@ -718,12 +733,36 @@ func TestUpdateFieldCompatSetArray(t *testing.T) {
 			resultType: emptyResult,
 		},
 		"ArrayNil": {
-			update: bson.D{{"$set", bson.D{{"v", bson.A{nil}}}}},
-			skip:   "https://github.com/FerretDB/FerretDB/issues/1662",
+			update:        bson.D{{"$set", bson.D{{"v", bson.A{nil}}}}},
+			skipForTigris: "tigris produce empty result because composites dataset is not applicable",
 		},
 		"EmptyArray": {
 			update: bson.D{{"$set", bson.D{{"v", bson.A{}}}}},
 		},
+		"ArrayNumbersAsc": {
+			update:        bson.D{{"$set", bson.D{{"v", bson.A{int32(42), int64(43), 45.5}}}}},
+			skipForTigris: "https://github.com/FerretDB/FerretDB/issues/1668",
+		},
+		"ArrayStringsDesc": {
+			update:        bson.D{{"$set", bson.D{{"v", bson.A{"c", "b", "a"}}}}},
+			skipForTigris: "https://github.com/FerretDB/FerretDB/issues/1816",
```

Comment:
This one should work for tigris, see https://github.com/FerretDB/FerretDB/blob/71ba4d0c578ba0777b9464e17f979610e2a42f88/integration/shareddata/composites.go#L133

Reply:
Yes it works on `ArrayStrings` :+1: , I just need to skip `ArrayDoubles`, `ArrayInt32s` and `ArrayRegexes`